package.json

```diff
@@ -76,7 +76,7 @@
     "@commitlint/config-conventional": "^17.4.3",
     "@doc-tools/transform": "^3.3.2",
     "@gravity-ui/eslint-config": "^2.0.0",
-    "@gravity-ui/page-constructor": "^4.0.0",
+    "@gravity-ui/page-constructor": "^4.7.0",
```

Comment:
is  another components are ok after this update?

Reply:
it looks so